### ISSUE TITLE
Several keybindings for one action?

### ISSUE THREAD

**Issue by for-coursera** [insanum/sncli#33]
Is is possible to have several alternate keybindings for one action? Like, `q`, `<Esc>`, `<F10>` for quit?

**Comment by swalladge** [comment]
Not currently. It could be possible, but easily become quite messy in the codebase...

What use cases are you thinking of where this would be useful?

**Comment by for-coursera** [comment]
> What use cases are you thinking of where this would be useful?

Just indulging to a person's habits? :) Hotkeys are mostly the muscular memories, so adding more options could save some additional seconds for some of us.

On the other hand, if I'm the first to ask, then it's not that badly needed, I presume.

**Comment by swalladge** [comment]
Yep this hasn't come up before. I'm trying to think of a use case that could
make it worthwhile attempting to implement it. Currently you can remap any of
the default keybindings to whatever you want, so if you find one of `q` or
`<esc>` or `<F10>` faster for quit, then you're free to rebind it. This has
obviously been enough so far.

Of course if you can come up with a neat, efficient method of implementing it,
feel free to submit a pull request. :)
